cycling/neware_processing_functions.py

```diff
@@ -181,15 +181,15 @@ def parse_record_header(position, nested=True):
             if "Vol" in stripped:
                 iter_dat = [
                     ("voltage","Vol"),
-                    ("current","Cur"),
+                    (Key.I,"Cur"),
                     ("capacity","Cap"),
                     ("time","Time"),
                     ("realtime","Realtime"),
                 ]
             elif "Voltage" in stripped:
                 iter_dat = [
                     ("voltage","Voltage"),
-                    ("current","Current"),
+                    (Key.I,Key.I),
```

Comment:
nope this is wrong. please roll back and don't touch these functions. they are rarely tested and they are not setup to detect these errors.